### PR TITLE
OCM-6955 | fix: provide better message for external auth creation

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-6955

### CODE REVIEW

File: cmd/create/externalauthprovider/cmd.go [hunk 1]
```diff
@@ -90,7 +90,9 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
-	r.Reporter.Infof("Successfully created an external authentication provider for cluster '%s'", cluster.ID())
+	r.Reporter.Infof("Successfully created an external authentication provider for cluster '%s'. "+
+		"It can take a while for the creation of an external authentication provider to become fully effective.",
```
Comment (philipwu08) [inline cmd/create/externalauthprovider/cmd.go:94]: "a while" doesn't give me much idea about how long to wait. Maybe try to give the user a better idea with wording like: "a few minutes", "up to 30 minutes", ...

Comment (chenz4027) [reply cmd/create/externalauthprovider/cmd.go:94]: Thanks @philipwu08 , I'll go with a few minutes then

Comment (chenz4027) [reply cmd/create/externalauthprovider/cmd.go]: @philipwu08 Updated, would you mind taking another look?

**Comment by oriAdler** [comment]
LGTM

File: cmd/create/externalauthprovider/cmd.go [hunk 2]
```diff
@@ -90,7 +90,9 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
-	r.Reporter.Infof("Successfully created an external authentication provider for cluster '%s'", cluster.ID())
+	r.Reporter.Infof("Successfully created an external authentication provider for cluster '%s'. "+
+		"It can take a few minutes for the creation of an external authentication provider to become fully effective.",
+		cluster.ID())
```
Comment (robpblake) [inline cmd/create/externalauthprovider/cmd.go:95]: @chenz4027 Should this be `clusterKey` and not `cluster.ID()` ?

Comment (chenz4027) [reply cmd/create/externalauthprovider/cmd.go:95]: Good catch @robpblake

Comment (chenz4027) [reply cmd/create/externalauthprovider/cmd.go]: Updated

**Pull request merged**